util_function_groups/column_operations.R

```diff
@@ -3,6 +3,219 @@
 source("~/Sites/gymnast/util.R", chdir = FALSE)
```

Comment:
We can't source util from the subfiles. That would get us into a recursive loop.